From d696645b147eaac5d5c90ff3dca672e52d89d7f0 Mon Sep 17 00:00:00 2001
From: Jerry Snitselaar <jsnitsel@redhat.com>
Date: Mon, 27 Apr 2020 12:16:47 -0700
Subject: [PATCH] man: Clean up libmandoc parser warnings

- Fix typo in Tss2_Tcti_Device_Init.3.in.
- Remove .RE macros that had no preceding .RS macro in Tss2_TctiLdr_Initialize.3.in.
  Replace .RE .sp with .LP.
- ' is a control character, format function names to be similar to
  other manpages, and use \(oq and \(cq for quotes instead in tss2-tctildr.7.in.

Signed-off-by: Jerry Snitselaar <jsnitsel@redhat.com>
---
 man/Tss2_TctiLdr_Initialize.3.in |  6 ++----
 man/Tss2_Tcti_Device_Init.3.in   |  2 +-
 man/tss2-tctildr.7.in            | 12 ++++++++----
 3 files changed, 11 insertions(+), 9 deletions(-)

diff --git a/man/Tss2_TctiLdr_Initialize.3.in b/man/Tss2_TctiLdr_Initialize.3.in
index 8e5fffaa247b..eb5ea1f8315f 100644
--- a/man/Tss2_TctiLdr_Initialize.3.in
+++ b/man/Tss2_TctiLdr_Initialize.3.in
@@ -65,8 +65,7 @@ libtss2-tcti-tabrmd.so.0
 libtss2-tcti-device.so.0
 .IP \[bu]
 libtss2-tcti-mssim.so.0
-.RE
-.sp
+.LP
 When the
 .I name
 string is neither NULL nor the empty string the implementation will attempt
@@ -81,8 +80,7 @@ name with the following permutations:
 libtss2-tcti-<name>.so.0
 .IP \[bu]
 libtss2-tcti-<name>.so
-.RE
-.sp
+.LP
 The
 .I config
 string is not interpreted by the TctiLdr init functions and is passed
diff --git a/man/Tss2_Tcti_Device_Init.3.in b/man/Tss2_Tcti_Device_Init.3.in
index 3cd2eed7fb0b..122ede1536bc 100644
--- a/man/Tss2_Tcti_Device_Init.3.in
+++ b/man/Tss2_Tcti_Device_Init.3.in
@@ -86,7 +86,7 @@ is returned if any parameters contain unexpected values.
 is returned if any parameters are NULL when they should not be.
 .B TSS2_TCTI_RC_BAD_CONTEXT
 is returned if the size of the provided
-.i tctiContext
+.I tctiContext
 is insufficient.
 .SH EXAMPLE
 TCTI initialization fragment:
diff --git a/man/tss2-tctildr.7.in b/man/tss2-tctildr.7.in
index a907aec0cd64..7432316ec6bb 100644
--- a/man/tss2-tctildr.7.in
+++ b/man/tss2-tctildr.7.in
@@ -10,13 +10,17 @@ instances.
 .SH DESCRIPTION
 The TCTI dynamic loading and initialization protocol requires a lot of
 boilerplate code. To reduce duplication the tss2-tctildr library adds the
-'Tss2_TctiLdr_Initialize', 'Tss2_TctiLdr_Initialize_Ex' and
-'Tss2_TctiLdr_Finalize' functions to abstract away the machinery required
+.BR Tss2_TctiLdr_Initialize (),
+.BR Tss2_TctiLdr_Initialize_Ex (),
+and
+.BR Tss2_TctiLdr_Finalize ()
+functions to abstract away the machinery required
 to load, initialize, and finalize a TCTI context.
 
 To assist in the discovery of TCTIs this library provides the
-'Tss2_TctiLdr_GetInfo' function. This function, paired with a 'free'
-function to free the memory allocated by 'GetInfo', provides a simple
+.BR Tss2_TctiLdr_GetInfo ()
+function. This function, paired with a \(oqfree\(cq
+function to free the memory allocated by \(oqGetInfo\(cq, provides a simple
 query interface for discovery of the available and default TCTIs
 available to the tss2-tctildr implementation
 
-- 
2.24.0

